Documentation/upgrades/upgrade_3_3.md

```diff
@@ -12,6 +12,17 @@ Before [starting an upgrade](#upgrade-procedure), read through the rest of this
 
 Highlighted breaking changes in 3.3.
 
+#### Change in `--auto-compaction-retention` flag
+
+Changed `--auto-compaction-retention` flag to [accept string values](https://github.com/coreos/etcd/pull/8563) with [finer granularity](https://github.com/coreos/etcd/issues/8503). Now that `--auto-compaction-retention` accepts string values, etcd configuration YAML file `log-output` field must be changed to `string` type. Previously, `--config-file etcd.config.yaml` can have `auto-compaction-retention: 24` field, now must be `auto-compaction-retention: "24"` or `auto-compaction-retention: "24h"`. If configured as `--auto-compaction-mode periodic --auto-compaction-retention "24h"`, the time duration value for `--auto-compaction-retention` flag must be valid for [`time.ParseDuration`](https://golang.org/pkg/time/#ParseDuration) function in Go.
+
+```diff
+# etcd.config.yaml
++auto-compaction-mode: periodic
+-auto-compaction-retention: 24
++auto-compaction-retention: "24h"
```

Comment:
I am debating `"24"` vs `"24h"` since `"h"`  suffix is not strictly needed. 
However, `"24h"` is more readable.

But, the breakage is that the yaml must be string instead of a int. Having the `"h"` suffix doesn't illustrate the breakage.

I am fine with either. However, I prefer `"24"` as it Illustrates the breakage better.

Reply:
Sure, maybe add it as a comment?

Reply:
a comment will be better.